common/src/main/java/com/wynntils/mc/mixin/ScreenMixin.java

```diff
@@ -20,13 +22,17 @@
 import org.spongepowered.asm.mixin.Final;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.Redirect;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(Screen.class)
-public abstract class ScreenMixin {
+public abstract class ScreenMixin implements TextboxScreen {
```

Comment:
I don't think you need to add `implements` to mixins.

Reply:
Ah, I see, that was not a vanilla interfance, but a way to get to the text input field. Nevermind.